index.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>chigozie ezeilo portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <a> <div class="wrapper">
+        <header>
+            <nav>
+                 <img src="./images/Logo Placeholder.svg" alt="" class="logo">
+                </a>
+                 <button class="menu"><i class="fafa-bars"></i></button>
+             </nav>
+        </header>
+        <section class="header-container">
+            <h1>Hey There. I'm chigozie</h1>
+            <h3>I'm a software developer</h3>
+            <p>i can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+
+                <ul class="social-container">
+                    <li><a href="#"><img src="./images/git.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/v.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/twitter.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/M.svg" alt=""></a></li> 
+                </ul>
+
+
+        </section>
+        <section class="work-container">
+            <h2 class="recent-work">My Recent Work</h2>
+            <div class="featured-card">
+                <img src="./images/featured.svg" alt="" class="featured">
+                <div class="multi-container">
+                    <h3 class="feature">Multi-Post Stories</h3>
```

Comment:
- [ ] It seems your project title is not visible for some reason 🤔. Please consider fixing this issue and check the screenshots below to get an idea of what I missing 👇🏽. _**Try changing the color**_

### _Your design_
![image](https://user-images.githubusercontent.com/43172164/199297701-64dce35b-d1f1-4849-984c-39ed7d54dcbc.png)

### _Figma design_
![image](https://user-images.githubusercontent.com/43172164/199297785-cb3bf2d8-2088-4704-9b3e-97ebc1c13d39.png)